easybuild/easyconfigs/p/PyTorch/PyTorch-1.6.0-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,307 @@
+    'ab37b266702557b819a598d1bf89946cba281a8d49224ce105280eb657d0c733',  # ideep-20200616.tar.gz
+    '4bc7dafaaaa55db55bb2654e043cc7f3fb35cd49d19c4efee2dbce1ab863c179',  # mkl-dnn-20200605.tar.gz
+    'c3bf8cc8091b24c43aa205a1f2037a518d249eb5997bea7f543b7fc880e80e21',  # onnx-20200531.tar.gz
+    'c7682e9007ddfd94072647abab3e89ffd9084089460ae47d67060974467b58bf',  # benchmark-20180525.tar.gz
+    '314cde420a7cf692bdb6877bc6af6bc514805f6cdb8bee90f32566ed08d94b1c',  # onnx-tensorrt-20190916.tar.gz
+    'f6c4dab91ae9a03b3019e7cab0572743afd0e1b6e75b97fcca50259c737c924e',  # psimd-20200517.tar.gz
+    'd13818a10a645d557202aa005606337b2a0cdf690e5655253800ab11abb0c076',  # pthreadpool-20200615.tar.gz
+    '13100c3deed300bbf16f87d8af3539f432462bfef9d38f0c7e3e387dc2e88676',  # PeachPy-20180218.tar.gz
+    '8cb5fae822077ca9cbc14dcc7bba9a3a35ad519284fc5169f9a176672c63860a',  # sleef-20190730.tar.gz
+    # checksum for original download of tbb-20181009.tar.gz from https://github.com/intel/tbb/archive
+    ('dc0a8d8d96cb8765782aa6ac1b509ad4db955d9bbb58fa5cc2265f0292756d72',
+     # checksum for download of tbb-20181009.tar.gz from https://github.com/01org/tbb/archive
+     'be111cf161b587812fa3b106fe550efb6f129b8b0b702fef32fac23af9580e5e'),
+    '9d78a64a4456eb0caeba8742080c39e07019114c1f4914f0e1ffa8f0a507c40f',  # tensorpipe-20200626.tar.gz
+    '5ca4e9091f3231d8ad8801862dc4e851c23af89c69141d27723157776f7291e7',  # libuv-20200419.tar.gz
+    '1c9f36f959ea5d7cf0d4ca678ec6bf60ef9ed16c556f97197aa018c7b1e5e21d',  # PyTorch-1.4.0_fix-system-pybind11.patch
+    'f3725477d554800a19a8d7f4ab300ab7b70da76b62328081c0ed1f8517c6a61c',  # PyTorch-1.6.0_fix-missing-source-dir.patch
+    'adf21e77dbc5e60fd46fe21fceb908633b0da7d4fc9b6a3ea7f7cfb0c4a44678',  # PyTorch-1.6.0_fix-test-cuda.patch
+    # PyTorch-1.6.0_fix-test-dataloader-fixed-affinity.patch
+    'a4208a46cd2098744daaba96cebb96cd91166f8fc616924315e05974bad80c67',
+    '37c19e286f4090115bc67989e537241f2988041ef5c3ff6d02d8c736d8703090',  # PyTorch-1.6.0_fix-test-tensorexpr.patch
+    # PyTorch-1.6.0_fix-test-missing-rpc-shutdown.patch
+    '9afd9b3a85a7b115e9535cf6a618d5dd137a69a5f36747d6290ac3bf123ab896',
+    '7d48c4ef7ae98a8a9e5eade3341ce18e41fa3fe4b868e9435efbfba849d1f977',  # PyTorch-1.6.0_fix-rpc-test-world-size.patch
+    'f0e09866e500a6ec96f7a0ad606d220960683901aa8dc491e245f9f8f4d89e45',  # PyTorch-1.6.0_fix-system-pybind11.patch
+]
+
+osdependencies = [OS_PKG_IBVERBS_DEV]
+
+builddependencies = [
+    ('CMake', '3.15.3'),
+    ('hypothesis', '4.44.2', versionsuffix),
+]
+
+dependencies = [
+    ('Ninja', '1.9.0'),  # Required for JIT compilation of C++ extensions
+    ('protobuf', '3.10.0'),
+    ('Python', '3.7.4'),
+    ('pybind11', '2.4.3', versionsuffix),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+    ('PyYAML', '5.1.2'),
+    ('MPFR', '4.0.2'),
+    ('GMP', '6.1.2'),
+    ('numactl', '2.0.12'),
+    ('FFmpeg', '4.2.1'),
+    ('gflags', '2.2.2'),
+    ('glog', '0.4.0'),
```

Comment:
Please remove these dependencies as it was already done in https://github.com/easybuilders/easybuild-easyconfigs/pull/11323 for `fosscuda`
```suggestion
```